mne/stats/cluster_level.py

```diff
@@ -1766,8 +1772,10 @@ def _validate_cluster_df(df: pd.DataFrame, dv_name: str, iv_name: str):
         )
     # check if the shape of the data is consistent
     if isinstance(inst, np.ndarray):
-        all_shapes = set(df[dv_name].map(lambda x: x.shape[1:]))  # first dim may vary
-    elif isinstance(inst, BaseEpochs):
+        all_shapes = set(
+            df[dv_name].map(lambda x: x.shape[1:])
+        )  # first dim may vary (participants or epochs)
+    elif isinstance(inst, (BaseEpochs | BaseTFR)):  # should include BaseTFR?
```

Comment:
I suspect this should be

```suggestion
    elif isinstance(inst, (BaseEpochs | EpochsTFR)):
```

Reply:
Yes

Reply:
yes there is a BaseTFR, but we don't actually want to include AverageTFR in this else branch.